tools/app_info.exs

```diff
@@ -1,7 +1,7 @@
 defmodule AppInfo do
   def name,    do: config |> Dict.get(:app)
   def version, do: config |> Dict.get(:version)
-  def exrm,    do: deps |> Dict.has_key?(:exrm)
+  def distillery,    do: deps |> Dict.has_key?(:distillery)
```

Comment:
We can get rid of all the superfluous whitespaces if you like (also from the surrounding functions.
I had the habit of `do` alignments in the past, but won't stick and enforce it anymore.